package.json

```diff
@@ -60,9 +59,7 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
-  "peerDependencies": {
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#v1.0.0"
-  },
+  "peerDependencies": {},
```

Comment:
why not peer?

Reply:
Was previously in the dev dependency. I wanted to update it to the latest so I removed and added but didn't notice it was added to the dependencies... Good catch and even better suggestion as it is really a peer dependency and not only dev. Thanks